README.md

```diff
@@ -86,6 +86,16 @@ The extension provides a quick command to export your AsciiDoc file as DocBook.
 
 **ℹ️ Note:** Only DocBook 5 is supported.
 
+### Paste Image
+
+The extension provides a quick command to paste an image from clipboard.
+
+* Create a screenshot thats in the clipboard
```

Comment:
Might be too specific? "Copy an image in your clipboard" ?

Reply:
Yes, great idea! This is the more generic approach. My usecase actually was: take a screenshot and put in in a adoc document. 
But put an image to clipboard is more versatile, should also work e.g. for remote sessions on other hosts.